### PR TITLE
Clear messages should not be present when in readonly mode #481

### PR DESCRIPTION
**Pull request by MBovtriukProvectus**
with read-only mode
topic list
Screenshot 2021-05-24 at 20 50 18
in topic 
Screenshot 2021-05-24 at 20 53 34

with out read-only mode
Screenshot 2021-05-24 at 20 51 28
in topic 
Screenshot 2021-05-24 at 20 55 18

### CODE REVIEW

File: kafka-ui-react-app/src/components/Topics/List/List.tsx [hunk 1]
```diff
@@ -147,6 +147,7 @@ const List: React.FC<Props> = ({
                   topic={topic}
                   deleteTopic={deleteTopic}
                   clearTopicMessages={clearTopicMessages}
+                  isReadOnly={isReadOnly}
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/List/List.tsx:150]: pls use context instead

File: kafka-ui-react-app/src/components/Topics/List/ListItem.tsx [hunk 2]
```diff
@@ -15,13 +15,15 @@ export interface ListItemProps {
...
   clearTopicMessages(topicName: TopicName, clusterName: ClusterName): void;
+  isReadOnly: boolean;
 }
 
 const ListItem: React.FC<ListItemProps> = ({
   topic: { name, internal, partitions },
   deleteTopic,
   clusterName,
   clearTopicMessages,
+  isReadOnly,
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/List/ListItem.tsx:26]: useContext

File: kafka-ui-react-app/src/components/Topics/List/List.tsx (hunk 1 above)
Comment (MBovtriukProvectus) [reply kafka-ui-react-app/src/components/Topics/List/List.tsx]: done

File: kafka-ui-react-app/src/components/Topics/List/ListItem.tsx (hunk 2 above)
Comment (MBovtriukProvectus) [reply kafka-ui-react-app/src/components/Topics/List/ListItem.tsx]: done

**Review by workshur — APPROVED** [review]

**Pull request merged**